blog.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            My Blog
+        </title>
+        <link rel="stylesheet" type="text/css" href="styles.css" />
+    </head>
+    <body>
+        <nav class="navbar">
+            <h1 class="logo"><a href="index.html">DG</a></h1>
+            <ul class="nav-list">
+              <li class="nav"><a href="index.html">Home</a></li>
+              <li class="nav"><a href="blog.html">Blog</a></li>
+              <li class="nav"><a href="portfolio.html">Portfolio</a></li>
+              <li class="nav"><a href="resume.html">Resume</a></li>
+              <li class="nav"><a href="contact.html">Contact</a></li>
+            </ul>
+        </nav>
+        <main>
+            <h1 class="page-title">MY BLOG</h1>
+            Hello World! This is my blog.
+        </main>
+        <footer>© 2023 Dhanvi Ganti | All Rights Reserved</footer>
+    </body>
```

Comment:
There should be a `<script>` tag somewhere here that links to your js file.

Reply:
Fixed! Please let me know if I have any other issues I need to fix, thank you so much for your help!

Reply:
Added the biog list as well for the individual htmls